components/command/package.go

```diff
@@ -37,7 +37,7 @@ func NewGenericPackageManager(
 	return packageManager
 }
 
-func (m *GenericPackageManager) Ensure(packageRef string, opts ...pulumi.ResourceOption) (*remote.Command, error) {
+func (m *GenericPackageManager) Ensure(packageRef string, customizer Customizer, opts ...pulumi.ResourceOption) (*remote.Command, error) {
```

Comment:
<!-- Suggestions propose improvements to the current subject. It’s important to be explicit and clear on what is being suggested and why it is an improvement. -->
**💬 suggestion**
```suggestion
func (m *GenericPackageManager) Ensure(packageRef string, customize Customizer, opts ...pulumi.ResourceOption) (*remote.Command, error) {
```